.github/workflows/nightly-cron.yml

```diff
@@ -0,0 +1,18 @@
+# Nightly Cron workflow for GitHub Actions
+name: "⏰ Nightly Cron"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  branches:
+    - main
+
+jobs:
+  nightly-cron:
```

Comment:
You should also reset the active streak for users who do not complete a puzzle within the 24 hours when running the nightly cron job!

Reply:
Thanks for the reminder! I totally forgot that and just added it